skeleton-typescript/index.html

```diff
@@ -16,7 +16,13 @@
     <script src="jspm_packages/system.js"></script>
     <script src="config.js"></script>
     <script>
-      System.import('aurelia-bootstrapper');
+      // ensure jQuery is laoded before other imports in you project
+      System.import('jquery')
+        .then(jQuery => {
+          window.jQuery = jQuery;
```

Comment:
i'm wondering what exactly is the case/issue, where global jQuery is required? I guess the code, that expects the global jQuery, isn't packaged as JS module (that declares jQuery as a dependency).